### PR TITLE
Added xcode-select fix to iOS export page

### PR DESCRIPTION
**Pull request by AdriaandeJongh**
This PR adds a troubleshooting section to the export to iOS page, and a section about a iPhone SDK path error with a relatively simple fix.

### CODE REVIEW

**Comment by AdriaandeJongh** [comment]
No idea why this failed the checks.

File: tutorials/export/exporting_for_ios.rst [hunk 1]
```diff
@@ -128,3 +128,36 @@ the export menu.
...
+Troubleshooting
+---------------
+
+xcode-select points at wrong SDK location
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+xcode-select is a tool that comes with XCode and among other things points at iOS development SDKs on your mac.
+If you have Xcode installed, opened it, agreed to the license agreement, and installed the command line tools, 
+xcode-select should point at the right location for the iPhone SDK. 
+If it somehow doesn't, Godot will fail exporting to iOS with an error that may looks like this:
```
Comment (AThousandShips) [inline tutorials/export/exporting_for_ios.rst:141]:
```suggestion
If it somehow doesn't, Godot will fail exporting to iOS with an error that may look like this:
```
Grammar error I missed

**Review by bruvzg — APPROVED** [review]
Please squash the commits into one, see [Pull request workflow](https://docs.godotengine.org/en/stable/contributing/workflow/pr_workflow.html#the-interactive-rebase).

**Comment by AdriaandeJongh** [comment]
> Please squash the commits into one, see [Pull request workflow](https://docs.godotengine.org/en/stable/contributing/workflow/pr_workflow.html#the-interactive-rebase).

sorry – I really f*cked up my GIT branches to the point where I have no idea how to fix it! The doc you link to doesn't describe the workflow for an accidental merge with master. Is there a way I could redo / recreate this branch, perhaps?

**Comment by AdriaandeJongh** [comment]
A hard reset to my original commit and a force push did the trick! That should probably be in the docs as well?

File: tutorials/export/exporting_for_ios.rst [hunk 2]
```diff
@@ -128,3 +128,36 @@ the export menu.
...
    * - Options / Application / Provisioning Profile UUID Release
      - ``GODOT_IOS_PROVISIONING_PROFILE_UUID_RELEASE``
+
+Troubleshooting
+---------------
+
+xcode-select points at wrong SDK location
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+xcode-select is a tool that comes with XCode and among other things points at iOS development SDKs on your mac.
```
Comment (mhilbrunner) [inline tutorials/export/exporting_for_ios.rst:138]:
```suggestion
xcode-select is a tool that comes with Xcode and among other things points at iOS SDKs on your Mac.
```

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Squashed and merged. Thanks!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2.

**Pull request merged**